src/views/v2/commerce/gateway/stripe/payment_element.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Buttons for Stripe
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/gateway/stripe/buttons.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ * @var bool $must_login [Global] Whether login is required to buy tickets or not.
+ */
+
+if ( $must_login ) {
+	return;
+}
+?>
+<form id="payment-form">
+	<div id="card-element">
+		<!-- Elements will create input elements here -->
+	</div>
+
+	<!-- We'll put the error messages in this element -->
+	<div id="card-errors" role="alert"></div>
+
+	<button id="tec-tc-gateway-stripe-checkout-button">Submit Payment</button>
+</form>
```

Comment:
Is there any way we can change the IDs here? Just to make sure we are namespacing it.

Reply:
@bordoni I looked around the documentation and couldn't find a way to change the `payment-form` ID, but the others are taken care of.